src/components/AccountDashboard.tsx

```diff
@@ -1,21 +1,82 @@
 import * as React from 'react'
 
-import { Transactions } from '@containers/Transactions'
-import { Transfer } from '@containers/Transfer'
-import { WalletInfo } from '@containers/WalletInfo'
+import { CurrencySelector } from '@containers/CurrencySelector'
 
-export const AccountDashboard: React.SFC = () => (
+import * as kagLogo from '@icons/kag-icon.svg'
+import * as kauLogo from '@icons/kau-icon.svg'
+import { RootState } from '@store'
+import { Currency, RootRoutes } from '@types'
+import * as sleeping from 'images/wally-sleeping.svg'
+import { connect } from 'react-redux'
+import { NavLink } from 'react-router-dom'
+
+const mapStateToProps = (state: RootState) => ({
+  currency: state.connections.currentCurrency,
+  balance: state.accounts.accountInfo.balance,
+})
+
+type Props = ReturnType<typeof mapStateToProps>
+
+const AccountDashboardPresentation: React.SFC<Props> = props => (
   <React.Fragment>
-    <div className='columns is-constant-height'>
-      <div className='column is-half'>
-        <WalletInfo />
-      </div>
-      <div className='column is-half'>
-        <Transfer />
-      </div>
+    <div className="container">
+      <CurrencySelector />
+      <section className="section has-text-centered">
+        <div className="level">
+          <div className="level-item">
+            <figure className="image is-128x128">
+              <img
+                src={props.currency === Currency.KAU ? kauLogo : kagLogo}
+                className="is-rounded"
+              />
+            </figure>
+          </div>
+        </div>
+        <div className="level">
+          <div className="level-item">
+            <h1
+              className={`title is-size-2 has-text-weight-bold ${
+                props.currency === Currency.KAU ? 'has-text-primary' : 'has-text-grey-light'
+              }`}
+            >{`${props.balance} ${props.currency}`}</h1>
+          </div>
+        </div>
+        <div className="level">
+          <div className="level-item">
+            <NavLink to={RootRoutes.dashboard + '/send'} className="button is-primary is-large">
```

Comment:
just a small gripe, but for consistency it'd be good to keep the interpolations going, eg. 
```typescript
to={`${RootRoutes.dashboard}/send`}
```

Reply:
Throughout pretty much everywhere I've used this syntax for putting together the different enum routes. Can change it, but I think it makes more sense when the DashboardRoutes are put into an enum.